src/Statistics/Logger/HttpStatisticsLogger.php

```diff
@@ -65,6 +65,19 @@ protected function findOrMakeStatisticForAppId($appId): Statistic
         return $this->statistics[$appId];
     }
 
+    protected function getUrl(): string
+    {
+        $action = [WebSocketStatisticsEntriesController::class, 'store'];
+
+        $url_override = config('websockets.statistics.base_url_override', null);
+
+        if ($url_override !== null) {
+            return $url_override.action($action, [], false);
```

Comment:
What is this supposed to do? 🤔 If `$url_override` is a static string, wouldn't this break?

Reply:
I'm not sure I understand your question. Basically it just overrides the base URL, for example I would set "http://caddy" as the base because that's the name of my web server's docker container. Localhost doesn't work for me because my websocket server is not in the same container and the APP_URL doesn't work for me because that's the external user facing URL. I need to specify an internal URL for the websocket server to reach my laravel app over http.

Reply:
I'm realizing maybe you're possibly misreading the `.` as a JS method call? That's a PHP concatenation 😛

I'd typically have spaces around the operator. Maybe I changed it because StyleCI complained, I don't remember.

Reply:
The signature for action is `function action($name, $parameters = [], $absolute = true)`, passing false for the 3rd param goes down to here https://github.com/laravel/framework/blob/6.x/src/Illuminate/Routing/RouteUrlGenerator.php#L77 which makes the base URL not get included in the output, so that it returns just the path without the base. Then we prefix that with the base URL override configured.